src/java.base/share/classes/java/security/spec/PSSParameterSpec.java

```diff
@@ -71,6 +71,11 @@
  *     SaltLength   -- 20
  *     TrailerField -- 1
  *
+ * <p>Its values are based on the default values in the ASN.1 encoding
+ * from PKCS#1 standard and may become obsolete as time progresses.
+ * Please do not rely on PSSParameterSpec.DEFAULT unless these values are
```

Comment:
Put `PSSParameterSpec.DEFAULT` in `{@code...}`?

Reply:
Yes, will do.